Update virtual-machines-sql-server-akv-next-steps.md

Max-Meng opened the pull request:
The current version mismatch the SQL logins by having misleading login name and description.
Also the T-SQL command to enable EKM provider and create provider is not necessary, because it is already done when we enable the AKV integration on the portal. 

sp_configure [show advanced options], 1;
GO
RECONFIGURE;
GO

-- Enable EKM provider
sp_configure [EKM provider enabled], 1;
GO
RECONFIGURE;

--create provider
CREATE CRYPTOGRAPHIC PROVIDER AzureKeyVault_EKM_Prov
FROM FILE = 'C:\Program Files\SQL Server Connector for Microsoft Azure Key Vault\Microsoft.AzureKeyVaultService.EKM.dll';
GO

azurecla commented:
Hi __@jurajsucik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
@jurajsucik - thanks very much for your contribution!
@kgremban - can you have a look at this PR? Juraj is a MSFT employee. The PR is currently in merge conflict, so it's probably best if you can review these and decide if they should be rolled into a different/internal PR. LMK and I can close this one after you have a chance to review. Thanks!

kgremban commented:
@BryanLa This branch is in conflict because I separately made an update similar to the one @jurajsucik requested. This PR can be closed. Thank you for the feedback, Juraj!

BryanLa commented:
@kgremban Done. Thanks again!

azurecla commented:
Hi __@jurajsucik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

BryanLa commented:
@jurajsucik - please ignore the bot message. The GitHub UI hung for a second, causing a brief reopen/close.

Max-Meng committed:
Update virtual-machines-sql-server-akv-next-steps.md

The current version mismatch the SQL logins by having misleading login name and description.
Also the T-SQL command to enable EKM provider and create provider is not necessary, because it is already done when we enable the AKV integration on the portal. 

sp_configure [show advanced options], 1;
GO
RECONFIGURE;
GO

-- Enable EKM provider
sp_configure [EKM provider enabled], 1;
GO
RECONFIGURE;

--create provider
CREATE CRYPTOGRAPHIC PROVIDER AzureKeyVault_EKM_Prov
FROM FILE = 'C:\Program Files\SQL Server Connector for Microsoft Azure Key Vault\Microsoft.AzureKeyVaultService.EKM.dll';
GO
